src/help/help_test.go

```diff
@@ -1,11 +1,23 @@
 package help
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func TestMain(t *testing.M) {
+	f, err := os.Create(".plzconfig")
+	if err != nil {
+		panic(err)
+	}
+	f.Close()
+
+	t.Run()
```

Comment:
`os.Exit(t.Run())`

you're losing the exit code

Reply:
What if it exits with a bad code? Then my test will fail. Wouldn't want that.